src/KernelMemory.Ecommerce.Sample.Api/Application/ProductIngestionCommand/ProductIngestionCommand.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Text.Json;
+using KernelMemory.Ecommerce.Sample.Api.Application.CsvReader;
+using KernelMemory.Ecommerce.Sample.Api.Domain;
+using Microsoft.KernelMemory;
+
+namespace KernelMemory.Ecommerce.Sample.Api.Application.ProductIngestionCommand;
+
+public sealed record ProductIngestionCommand(Stream ProductsFileStream) : ICommand;
+
+public sealed class IngestProductsCommandHandler(
```

Comment:
@demid-ns rename to ProductIngestionCommandHandler?

Reply:
Good catch! Fixed